### PR TITLE
[16]Render the related product list with JS

### PR DESCRIPTION
**Pull request by ChabuChabu**
## Task: [16]Render the related product list with JS
- `Wrote a javaScript program which rendered product related information onthe  console
productList

### CODE REVIEW

File: submissions/chabuChabu/javaScript/index.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    imageUrl: "../html/images/product-03.jpg",
+    productPrice: "$350",
+   },
+   {
+    productName: "Red High-heeled shoes",
+    imageUrl: "../html/images/product-04.jpg",
+    productPrice: "$230",
+   }
+]
+console.log(productList[0].productName+" with "+productList[0].imageUrl+" is sold for "+productList[0].productPrice);
```
Comment (ToluAkin) [inline submissions/chabuChabu/javaScript/index.js:18]: @ChabuChabu You are meant to loop through the array and make an output not one after the other.

**Review by ToluAkin — CHANGES_REQUESTED** [review]

File: submissions/chabuChabu/javaScript/index.js (hunk 1 above)
Comment (adaugochi) [inline submissions/chabuChabu/javaScript/index.js:18]: Do all this inside a loop

**Review by adaugochi — CHANGES_REQUESTED** [review]

File: submissions/chabuChabu/javaScript/index.js (hunk 1 above)
Comment (ChabuChabu) [reply submissions/chabuChabu/javaScript/index.js]: Code changed

Comment (ChabuChabu) [reply submissions/chabuChabu/javaScript/index.js]: Done

File: submissions/chabuChabu/javaScript/index.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    imageUrl: "../html/images/product-03.jpg",
+    productPrice: "$350",
+   },
+   {
+    productName: "Red High-heeled shoes",
+    imageUrl: "../html/images/product-04.jpg",
+    productPrice: "$230",
+   }
+]
+for(i = 0;i < productList.length;i++){
```
Comment (ToluAkin) [inline submissions/chabuChabu/javaScript/index.js:18]: @ChabuChabu `i` is not declared a variable.

**Review by ToluAkin — CHANGES_REQUESTED** [review]

File: submissions/chabuChabu/javaScript/index.js (hunk 2 above)
Comment (ChabuChabu) [reply submissions/chabuChabu/javaScript/index.js]: Commet addressed

File: submissions/chabuChabu/javaScript/index.js [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+   {
+    productName: "Red High-heeled shoes",
+    imageUrl: "../html/images/product-04.jpg",
+    productPrice: "$230",
+   }
+]
+for(var i = 0;i < productList.length;i++){
+   console.log(productList[i].productName+" with "+productList[i].imageUrl+" is sold for "+productList[i].productPrice);
+}
+
```
Comment (adaugochi) [inline submissions/chabuChabu/javaScript/index.js:21]: remove the spaces below

Comment (ChabuChabu) [reply submissions/chabuChabu/javaScript/index.js]: Removed

**Review by adaugochi — APPROVED** [review]

**Pull request merged**